ruby-cda.gemspec

```diff
@@ -13,10 +13,10 @@ Gem::Specification.new do |s|
   s.files       = `git ls-files lib`.split($/)
   s.homepage    = 'https://github.com/niquola/ruby-cda'
 
-  s.add_dependency 'activesupport', '4.2.7'
+  s.add_dependency 'activesupport', '~> 5'
```

Comment:
could also have an even more relaxed version of `'>= 4.2'` or `'>= 5.0'` or something (thinking of when we upgrade to Rails 6)